conan/solo5/0.3.1/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+from conans import ConanFile,tools,AutoToolsBuildEnvironment
```

Comment:
you are not using the AutoToolsBuildEnvironment :)

Reply:
good point, will fix